aries-site/src/layouts/content/Example.js

```diff
@@ -93,6 +94,7 @@ export const Example = ({
   ...rest
 }) => {
   const theme = React.useContext(ThemeContext);
+  const viewport = React.useContext(ResponsiveContext);
```

Comment:
Is there a reason we are using `viewport` as opposed to `size` like we usually have been doing?

Reply:
size was already declared as a variable elsewhere, line 104 I believe

Reply:
What about destructing that line to be: 
```
const {
    size: { medium, small },
  } = defaultProps.theme.global;
```
Then, we can reserve `size` for ResponsiveContext you're adding

Reply:
or maybe better:
```
  const { medium, small } = defaultProps.theme.global.size;
```

Reply:
👍